tests/operator_test.go

```diff
@@ -2476,30 +2478,52 @@ spec:
 	})
 
 	Context("[rfe_id:4356]Node Placement", func() {
+		atLeastOnePendingPodShouldExistInDeployment := func(deploymentName string) error {
```

Comment:
Thanks
If possible please don't inline functions, 
define them at the end of the file
(for both functions)